### PR TITLE
#239 Added tag and environment args to deploy.sh

### PR DESCRIPTION
**Pull request by carlosmiranda**
#239: Added tag and environment arguments to `deploy.sh`. Moved deploy script invocation in `.rultor.yml` from `deploy` to `release`, environment is hard-coded for the meantime (see https://github.com/timelesslounge/timelessis/issues/239#issuecomment-464585020).

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#295](https://github.com/timelesslounge/timelessis/issues/295) is now in scope, role is `REV`

**Comment by 0crat** [comment]
This pull request [#295](https://github.com/timelesslounge/timelessis/issues/295) is assigned to @vladarefiev[/z](https://www.0crat.com/u/vladarefiev), here is [why](https://www.0crat.com/footprint/CF9LG1YCA/3bdd718d-6bfb-4f0f-95e4-ec09e31fc0d2); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @emilianodellacasa[/z](https://www.0crat.com/u/emilianodellacasa) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

**Comment by vladarefiev** [comment]
@rultor merge

File: .rultor.yml [hunk 1]
```diff
@@ -24,9 +24,9 @@ merge:
...
   script: |-
-    chmod +x ./deploy.sh
-    ./deploy.sh
+    echo "Deploy not supported"
+    exit -1
 release:
   script: |-
-    echo "There is nothing to release"
-    exit -1
+    chmod +x ./deploy.sh $tag staging
```
Comment (emilianodellacasa) [inline .rultor.yml]: @carlosmiranda Could you please rename this script as release.sh? To avoid confusion

**Comment by emilianodellacasa** [comment]
@carlosmiranda Just a simple comment, please check

**Comment by carlosmiranda** [comment]
@emilianodellacasa done, please check

**Comment by emilianodellacasa** [comment]
@rultor merge

**Comment by 0crat** [comment]
Job was finished in 22 hours, bonus for fast delivery is possible (see [§36](http://www.zerocracy.com/policy.html#36))

**Comment by 0crat** [comment]
Job [#295](https://github.com/timelesslounge/timelessis/issues/295) is not in the agenda of @vladarefiev[/z](https://www.0crat.com/u/vladarefiev), can't inspect

**Comment by 0crat** [comment]
The job [#295](https://github.com/timelesslounge/timelessis/issues/295) is now out of scope

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 point(s) just awarded to @emilianodellacasa[/z](https://www.0crat.com/u/emilianodellacasa)

**Comment by ypshenychka** [comment]
@0crat status

**Comment by 0crat** [comment]
> @0crat status ([here](https://github.com/timelesslounge/timelessis/issues/295#issuecomment-465518053))

@ypshenychka This is what I know about this job in [CF9LG1YCA](https://www.0crat.com/p/CF9LG1YCA), as in [§32](http://www.zerocracy.com/policy.html#32):

  * The job [#295](https://github.com/timelesslounge/timelessis/issues/295) is not in scope
  * The job is waiting quality review verdict by @ypshenychka[/z](https://www.0crat.com/u/ypshenychka) for 33min
  * @vladarefiev[/z](https://www.0crat.com/u/vladarefiev) will receive 20 minutes after quality review
  * Job [footprint](https://www.0crat.com/footprint/CF9LG1YCA?q=%7Bjob%3A%27gh%3Atimelesslounge%2Ftimelessis%23295%27%2Ctype%3A%7B%24not%3A%2F%5ENotify%2F%7D%7D) (restricted area)

**Comment by ypshenychka** [comment]
@vladarefiev According to our [QA Rules](http://www.zerocracy.com/policy.html#42):
>The code reviewer found at least three problems in the code.
Comments were mostly about design problems, not cosmetic issues.

No issues were found during code review. 
Please confirm that you'll try to find at least three major problems while future reviews.

**Comment by vladarefiev** [comment]
@ypshenychka Confirm. quality bad

**Comment by ypshenychka** [comment]
@vladarefiev thanks

**Comment by ypshenychka** [comment]
@0crat quality bad

**Comment by 0crat** [comment]
Quality is low, no payment, see [§31](http://www.zerocracy.com/policy.html#31)

**Comment by 0crat** [comment]
Quality review completed: +8 point(s) just awarded to @ypshenychka[/z](https://www.0crat.com/u/ypshenychka)

**Pull request merged**